backend/docs/plantuml/sequenceFileSharingPrepro.puml

```diff
@@ -0,0 +1,26 @@
+@startuml
+    participant "Processing Pipeline" as pipeline #LightGreen
+    participant "Backend" as backend #Orange
+    participant "S3 / Object Storage" as s3 #LightGray
+    database "Database" as DB
+
+    pipeline -> backend: request entries needing processing
+    backend -> DB: fetch entries + file IDs
```

Comment:
By file ID do you mean file name?

Reply:
I've changed it to "fetch entries (including file IDs and file names)", does that clarify it?